webapp/voting-webapp/application/README.md

```diff
@@ -16,27 +16,27 @@ The application architecture is made of 5 components (see diagram below):
 
 ## Resources
 
-The resources in this directory aren't going to launch experiments in StormForge Optimize in and off themselves. For that, see [locust-metrics](../locust-metrics). Nevertheless, here is a short description for each of them.
+The resources in this directory aren't going to launch experiments in StormForge Optimize Pro in and of themselves. For that, see [locust-metrics](../locust-metrics). Nevertheless, here is a short description for each of them.
```

Comment:
probably just `lucust` now…

Reply:
Fixed in https://github.com/thestormforge/examples/pull/77/commits/cd201e8f085af6da53edc1436f821be40a6f3997